clang/test/dpct/cmake_migration/case_041/expected.txt

```diff
@@ -11,7 +11,7 @@ target_link_options( quda PUBLIC )
 target_link_options(quda PUBLIC  )
 
 target_link_options(nvcv_util_compat PUBLIC
-    
+    # DPCT3002:1: '-static-libstdc++' is not supported together with '-fsycl' when using icpx compiler. You may need to manually fix it.
```

Comment:
```suggestion
    # DPCT3002:1: '-static-libstdc++' is removed because it is not supported to be used together with option '-fsycl' of SYCL compiler. You may need to check the build options.
```

Reply:
Okay